### DISCUSSION TITLE
Microsoft Remote Desktop is being renamed to Windows App

### DISCUSSION THREAD

**Discussion by mrtysn** [Homebrew/discussions#5622] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.23
ORIGIN: https://github.com/Homebrew/brew
HEAD: 42c11a4dedf352229271593de831161071603c66
Last commit: 3 days ago
Core tap JSON: 20 Sep 05:49 UTC
Core cask tap JSON: 20 Sep 05:49 UTC
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: code
HOMEBREW_MAKE_JOBS: 12
Homebrew Ruby: 3.3.4 => /opt/homebrew/Library/Homebrew/vendor/portable-ruby/3.3.4_1/bin/ruby
CPU: dodeca-core 64-bit arm_blizzard_avalanche
Clang: 16.0.0 build 1600
Git: 2.39.5 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 8.7.1 => /usr/bin/curl
macOS: 14.7-arm64
CLT: 16.0.0.0.1.1724870825
Xcode: N/A
Rosetta 2: false
```

### Output of `brew doctor`

```shell
Warning: Some installed casks are deprecated or disabled.
You should find replacements for the following casks:
  cakebrew
  meld

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  cowsay
```

### Description of issue

The `Microsoft Remote Desktop` application is being renamed to `Windows App`. If you follow the link for the client from the [Microsoft website](https://learn.microsoft.com/en-us/windows-server/remote/remote-desktop-services/clients/remote-desktop-mac), the app it redirects to is now called [Windows App](https://apps.apple.com/us/app/windows-app/id1295203466).

Should the respective [cask](https://formulae.brew.sh/cask/microsoft-remote-desktop) be renamed and the [cask code](https://github.com/Homebrew/homebrew-cask/blob/f8e1cd3a27f8190a0e6487fa8b8ce3f311975c2e/Casks/m/microsoft-remote-desktop.rb) be updated as well? How are such changes usually handled?

**Comment by SMillerDev** [comment]
> Should the respective [cask](https://formulae.brew.sh/cask/microsoft-remote-desktop) be renamed and the [cask code](https://github.com/Homebrew/homebrew-cask/blob/f8e1cd3a27f8190a0e6487fa8b8ce3f311975c2e/Casks/m/microsoft-remote-desktop.rb) be updated as well? How are such changes usually handled?

Yeah, probably. https://docs.brew.sh/Updating-Software-in-Homebrew

**Comment by daeho-ro** [comment, marked as answer]
It is applied on last week.
- https://github.com/Homebrew/homebrew-cask/pull/186284
